webrtc.html

```diff
@@ -4166,7 +4166,7 @@ <h2 id="sec.stats-model">Statistics Model</h2>
 
       <dl class="idl" title="partial interface RTCPeerConnection">
         <dt>Promise&lt;RTCStatsReport&gt; getStats(
-          optional MediaStreamTrack? selector)</dt>
+          optional MediaStreamTrack? selector = null)</dt>
```

Comment:
But you're testing `DOMString`. When I remove `nullable` from `getStats` in WebIDL like this:

```webidl
Promise<RTCStatsReport> getStats (optional MediaStreamTrack selector)
```

and call `pc.getStats(null)` then I get:

    TypeError: Argument 1 of mozRTCPeerConnection.getStats is not an object., line 56